lib/rails_db/connection.rb

```diff
@@ -2,7 +2,11 @@ module RailsDb
   module Connection
 
     def connection
-      ActiveRecord::Base.connection
+      if RailsDb.database_role.present?

```

Comment:
is it possible to test somehow? at least just to cover with spec?

sorry for late review, just realised I've many open PRs